### PR TITLE
Add Core Usage Metrics Components to Storybook

### PR DESCRIPTION
**Pull request by Monkeychip**
This PR is a follow up of the Core Usage Metrics project, released in 1.4.  For that project, we created 3 new components.  This PR adds those components to Storybook.

**Additionally, this PR:**
- Changes the name of the component `HttpRequestsBarChartSmall` to `HttpRequestsBarChartSimple`.  This makes more sense when viewing from Storybook. See screenshot.

- My prettier configuration automatically reformatted the `http-requests-bar-chart.stories.js` file.  I agree with the formatting changes, and so am committing them in this PR.

- I double-checked that the renamed component still shows on the UI as it did before.    

Here is a gif of the 3 new components on Storybook.

_Also, I believe this should be a 1.4.1 milestone, but let me know if that's incorrect._

### CODE REVIEW

File: ui/app/components/selectable-card.js [hunk 1]
```diff
@@ -2,16 +2,16 @@ import Component from '@ember/component';
...
- * SelectableCard components are card-like components that display a title, total, subtotal, and anything after they yield.
+ * SelectableCard components are card-like components that display a title, total, subtotal, and anything after the yield.
  * They are designed to be used in containers that act as flexbox or css grid containers.
  *
  * @example
  * ```js
  * <SelectableCard @cardTitle="Tokens" @total={{totalHttpRequests}} @subText="Total" @gridContainer={{gridContainer}}/>
  * ```
- * @param cardTitle='' {String} - cardTitle displays the card title
+ * @param cardTitle=null {String} - cardTitle displays the card title
```
Comment (chelshaw) [inline ui/app/components/selectable-card.js]: Is there a reason this changed? Seems like a type of `string` should have default emptystring, not null.

Comment (Monkeychip) [reply ui/app/components/selectable-card.js]: Good question.  If I didn't write in null I got a bunch of crazy characters.  Looking at other storybook examples, it looks like if you want a blank string, null is what is expected.

File: ui/stories/selectable-card-container.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+## SelectableCardContainer
+SelectableCardContainer components are used to hold SelectableCard components.  They act as a CSS grid container, and change grid configurations based on the boolean of @gridContainer.
+
+**Params**
+
+| Param | Type | Default | Description |
+| --- | --- | --- | --- |
+| counters | <code>Object</code> | <code></code> | Counters is an object that returns total entities, tokens, and an array of objects with the total https request per month. |
+| gridContainer | <code>Boolean</code> | <code>false</code> | gridContainer is optional.  If true, it's telling the container it will have a nested CSS grid. const MODEL = {  totalEntities: 0,  httpsRequests: [{ start_time: '2019-04-01T00:00:00Z', total: 5500 }],  totalTokens: 1, }; |
```
Comment (noelledaley) [inline ui/stories/selectable-card-container.md]: is this `const MODEL = {  totalEntities: 0,  httpsRequests: [{ start_time: '2019-04-01T00:00:00Z', total: 5500 }],  totalTokens: 1, }; ` supposed to be there?

Comment (Monkeychip) [reply ui/stories/selectable-card-container.md]: I debated on this one.  I'll remove it.  In the component's JS file it's helpful, but not in the storybook md file.

**Comment by noelledaley** [comment]
thank you for adding these to storybook! this is great.

while reviewing this PR, i noticed 2 things from playing with the `gridContainer` prop on `SelectableCard` and `SelectableCardContainer`:

the first is that the `gridContainer` prop doesn't change the view of the `SelectableCard` (or at least, not that i can tell). so maybe this prop should be removed from the knobs section on the `SelectableCard` page in Storybook? since it seems like that prop is only relevant when the component itself is rendered in context (i.e. inside a `SelectableCardContainer`). totally up to you, though.

the second is that toggling `gridContainer` breaks the layout of the `SelectableCard`s when the component is rendered in isolation (i.e. in Storybook, and without all the css from the actual Metrics page). i don't think this needs to change in the scope of this PR, but it's something to keep in mind if/when we use this component in other places -- the component will need to be refactored a bit to work in various settings.

you can see examples of this here:
storybook

anyway, just sharing these observations here for posterity. definitely not blockers, so i'll go ahead and approve!

**Comment by Monkeychip** [comment]
@noelledaley These are great comments, thank you.  I went ahead and removed `gridContainer` knob from the `selectable-card`.

**Pull request merged**